changelog/28989.txt

```diff
@@ -0,0 +1,3 @@
+```release-note:bug
+secret/db: Update static role rotation to generate new password if retried password fails.
```

Comment:
Suggest being more explicit in our description of the change
```suggestion
secret/db: Update static role rotation to generate a new password after 2 failed attempts.
```

Reply:
Updated